<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite hostname="DESKTOP-TCJEROD" failures="0" tests="5" name="testRunner.DsalgoRunner" time="33.670" errors="1" timestamp="2025-05-09T13:48:13 PDT" skipped="0">
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="5.634"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="7.036"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="8.703"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="5.303"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="6.994">
    <error message="element click intercepted: Element &lt;input type=&quot;submit&quot; value=&quot;Login&quot;&gt; is not clickable at point (1153, 127). Other element would receive the click: &lt;iframe scrolling=&quot;no&quot; frameborder=&quot;0&quot; src=&quot;https://gc.kis.v2.scr.kaspersky-labs.com/E3E8934C-235A-4B0E-825A-35A08381A191/kpm/kpm_promo_balloon.html?cssSrc=https%3A%2F%2Fgc.kis.v2.scr.kaspersky-labs.com%2FE3E8934C-235A-4B0E-825A-35A08381A191%2Fkpm%2Ftooltip.css&quot; style=&quot;border: 0px; height: 50px; width: 48px; left: 1121px; top: 102px; z-index: 2147483647; position: absolute;&quot;&gt;&lt;/iframe&gt;
=======
<testsuite hostname="LAPTOP-2KQV9F30" failures="4" tests="16" name="testRunner.DsalgoRunner" time="146.873" errors="1" timestamp="2025-05-09T17:31:41 CDT" skipped="0">
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="9.627">
    <failure message="BUG: Expected message &#039;Submission Successful&#039; but got: &#039;Error occurred during submission&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: BUG: Expected message 'Submission Successful' but got: 'Error occurred during submission' expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepDefinitions.ArraysStep.the_user_gets_success_message_submission_successful_in_the_console(ArraysStep.java:299)
at ✽.The user gets success message Submission successful in the console(file:///C:/Workspace/Ds-Algo2025/src/test/resources/Feature/Arrays.feature:122)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="10.510"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="9.958"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="8.834"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="9.550">
    <failure message="BUG: Expected message &#039;Error occurred&#039; but got: &#039;No tests were collected&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: BUG: Expected message 'Error occurred' but got: 'No tests were collected' expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepDefinitions.ArraysStep.the_user_gets_error_message_error_occurred_during_submission_in_the_console(ArraysStep.java:280)
at ✽.The user gets error message Error occurred during submission in the console(file:///C:/Workspace/Ds-Algo2025/src/test/resources/Feature/Arrays.feature:140)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="8.513"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="9.930"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="9.236">
    <failure message="BUG: Expected message &#039;Submission Successful&#039; but got: &#039;No tests were collected&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: BUG: Expected message 'Submission Successful' but got: 'No tests were collected' expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepDefinitions.ArraysStep.the_user_gets_success_message_submission_successful_in_the_console(ArraysStep.java:299)
at ✽.The user gets success message Submission successful in the console(file:///C:/Workspace/Ds-Algo2025/src/test/resources/Feature/Arrays.feature:146)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="8.782"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="8.847">
    <failure message="BUG: Expected message &#039;Submission Successful&#039; but got: &#039;Some Tests failed. Please review code&#039; expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: BUG: Expected message 'Submission Successful' but got: 'Some Tests failed. Please review code' expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertTrue(Assert.java:57)
at stepDefinitions.ArraysStep.the_user_gets_success_message_submission_successful_in_the_console(ArraysStep.java:299)
at ✽.The user gets success message Submission successful in the console(file:///C:/Workspace/Ds-Algo2025/src/test/resources/Feature/Arrays.feature:75)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="13.587"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="9.070"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="3.872">
    <error message="no such window: target window already closed
from unknown error: web view not found
>>>>>>> main
  (Session info: chrome=136.0.7103.93)
Build info: version: &#039;4.29.0&#039;, revision: &#039;5fc1ec94cb&#039;
System info: os.name: &#039;Windows 11&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;23.0.2&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
<<<<<<< HEAD
Command: [06a9acb442013ee3f6f899f3e8ed2a85, clickElement {id=f.8E31CB110472166E62ED75A6E5E2E1F0.d.ED494A5256C4A6B58924513C97583E89.e.13}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 136.0.7103.93, chrome: {chromedriverVersion: 136.0.7103.92 (cb81a4cc5087..., userDataDir: C:\Users\Kruthi\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:61947}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:61947/devtoo..., se:cdpVersion: 136.0.7103.93, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (06a9acb442013ee3f6f899f3e8ed2a85)] -&gt; xpath: //input[@value=&#039;Login&#039;]]
Session ID: 06a9acb442013ee3f6f899f3e8ed2a85" type="org.openqa.selenium.ElementClickInterceptedException">
      <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <input type="submit" value="Login"> is not clickable at point (1153, 127). Other element would receive the click: <iframe scrolling="no" frameborder="0" src="https://gc.kis.v2.scr.kaspersky-labs.com/E3E8934C-235A-4B0E-825A-35A08381A191/kpm/kpm_promo_balloon.html?cssSrc=https%3A%2F%2Fgc.kis.v2.scr.kaspersky-labs.com%2FE3E8934C-235A-4B0E-825A-35A08381A191%2Fkpm%2Ftooltip.css" style="border: 0px; height: 50px; width: 48px; left: 1121px; top: 102px; z-index: 2147483647; position: absolute;"></iframe>
=======
Command: [617c117718df0344fcf3885cce333aa4, findElement {value=//input[@value=&#039;Login&#039;], using=xpath}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 136.0.7103.93, chrome: {chromedriverVersion: 136.0.7103.92 (cb81a4cc5087..., userDataDir: C:\Users\chand\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50836}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50836/devtoo..., se:cdpVersion: 136.0.7103.93, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 617c117718df0344fcf3885cce333aa4" type="org.openqa.selenium.NoSuchWindowException">
      <![CDATA[org.openqa.selenium.NoSuchWindowException: no such window: target window already closed
from unknown error: web view not found
>>>>>>> main
  (Session info: chrome=136.0.7103.93)
Build info: version: '4.29.0', revision: '5fc1ec94cb'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
<<<<<<< HEAD
Command: [06a9acb442013ee3f6f899f3e8ed2a85, clickElement {id=f.8E31CB110472166E62ED75A6E5E2E1F0.d.ED494A5256C4A6B58924513C97583E89.e.13}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 136.0.7103.93, chrome: {chromedriverVersion: 136.0.7103.92 (cb81a4cc5087..., userDataDir: C:\Users\Kruthi\AppData\Loc...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:61947}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:61947/devtoo..., se:cdpVersion: 136.0.7103.93, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (06a9acb442013ee3f6f899f3e8ed2a85)] -> xpath: //input[@value='Login']]
Session ID: 06a9acb442013ee3f6f899f3e8ed2a85
=======
Command: [617c117718df0344fcf3885cce333aa4, findElement {value=//input[@value='Login'], using=xpath}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 136.0.7103.93, chrome: {chromedriverVersion: 136.0.7103.92 (cb81a4cc5087..., userDataDir: C:\Users\chand\AppData\Loca...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:50836}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:50836/devtoo..., se:cdpVersion: 136.0.7103.93, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: 617c117718df0344fcf3885cce333aa4
>>>>>>> main
at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:501)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:485)
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:215)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
<<<<<<< HEAD
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:223)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:76)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy39.click(Unknown Source)
at pageObject.LoginPF.login_button(LoginPF.java:97)
at pageObject.LoginPF.clickLoginwithoutusername(LoginPF.java:120)
at stepDefinitions.LoginSteps.the_user_leave_the_username_field_empty_and_click_login(LoginSteps.java:72)
at ✽.The user leave the username field empty and click login(file:///C:/Users/Kruthi/dsAlgo2025/Ds-Algo2025/src/test/resources/Feature/Login.feature:14)
]]>
    </error>
  </testcase> <!-- runScenario -->
=======
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:368)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:362)
at stepDefinitions.ArraysStep.the_user_is_in_home_page_after_sign_in(ArraysStep.java:47)
at ✽.The User is in home page after sign in(file:///C:/Workspace/Ds-Algo2025/src/test/resources/Feature/Arrays.feature:5)
]]>
    </error>
  </testcase> <!-- runScenario -->
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="8.400"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="9.326"/>
  <testcase classname="testRunner.DsalgoRunner" name="runScenario" time="8.831"/>
>>>>>>> main
</testsuite> <!-- testRunner.DsalgoRunner -->
